edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -24,8 +25,9 @@ def __init__(self, log="BaseParser"):
         self.ConditionalStack = []
         self.RootPath = ""
```

Comment:
Is this Workspace?  Is this abstracted from Workspace on purpose?

Reply:
self.RootPath? Yes it's effectively RootPath.